src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -17,10 +17,11 @@
 
     public static final String COMMAND_WORD = "delete";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "-"
```

Comment:
Please remember Object dash Command. If in doubt ask the group before making such changes to avoid unnecessary work like here.

Also note that in the application, the "DeleteCommand" is effectively an "DeletePersonCommand". Please do not be confused and think it is a generic class for adding items. It is not; these commands should be separate.

The message usage should be separate as well. "delete-pat|appt" is messy and hard to read as a user; each command class should only contain message usage for their own command.